contracts/stellar-axelar-gateway/Cargo.toml

```
@@ -2,7 +2,7 @@
 name = "stellar-axelar-gateway"
 version = "0.2.3"
 edition = { workspace = true }
-description = "Contract related to Stellar Axelar Gateway."
+description = "AxelarGateway contract, which serves as the gateway for cross-chain contract calls and token transfers within the Axelar network."
```

Please pick one or the other for all descriptions
```suggestion
description = "AxelarGateway contract, responsible for serving as the gateway for cross-chain contract calls and token transfers within the Axelar network."
```